dist/rudder-sdk-js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "rudder-sdk-js",
-  "version": "1.2.15",
+  "version": "1.2.18",
```

Comment:
Should not we change to 1.3.0 as event filtering is a new feature?

Reply:
Yeah, we should. We usually bump the version once the changes are merged to production.